apps/transmitmedia.cpp

```diff
@@ -805,15 +805,21 @@ class UdpCommon
 
         bool is_multicast = false;
 
-        if ( attr.count("multicast") )
+        if (attr.count("multicast"))
         {
+            // FIXME: Provide support for IPv6 multicast
```

Comment:
I would rather create a FR on GitHub than leave this comment in the code.
Otherwise, it goes to active alerts of LGTM.

Reply:
Ok, I'll change this to XXX so that the place is remembered.